src/api.js

```diff
@@ -11,11 +11,17 @@ export const getCategories = () => {
     })
 }
 
-export const getReviews = () => {
-    return gameApi.get('/reviews')
+export const getReviews = (category) => {
```

Comment:
This definitely works but you could do away with the if logic by using axios params here (a second arguement in the get request). Don't worry about changing this now though unless you want to as your solution is absolutely fine!